src/internal/m365/collection/drive/collection.go

```diff
@@ -265,7 +265,10 @@ func (oc *Collection) getDriveItemContent(
 
 		// Skip big OneNote files as they can't be downloaded
 		if clues.HasLabel(err, graph.LabelStatus(http.StatusServiceUnavailable)) &&
-			oc.scope == CollectionScopePackage && *item.GetSize() >= MaxOneNoteFileSize {
+			// TODO: We've removed the file size check because it looks like we've seen persistent
+			// 503's with smaller OneNote files also.
+			// oc.scope == CollectionScopePackage && *item.GetSize() >= MaxOneNoteFileSize {
+			oc.scope == CollectionScopePackage {
```

Comment:
The fault name still in use is `SkipBigOneNote`. Maybe we can change to reflect this change.

Reply:
I thought about that - will address when we figure out the final change here.